.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -19,36 +12,26 @@ feel free to remove what't not applicable.
 - [ ] Any commands are available via the command palette.
 - [ ] Preferences and keybindings (if any) are listed in the menu and the command palette, and open in split view.
 - [ ] If my package is a syntax it doesn't also add a color scheme. ***
-- [ ] I have read the [style guide](https://github.com/wbond/package_control_channel/?tab=readme-ov-file#style-guide)
 - [ ] I use [.gitattributes][3] to exclude files from the package: images, test files, sublime-project/workspace.
 
+[1]: https://docs.sublimetext.io/guide/package-control/submitting.html
+[2]: https://semver.org
```

Comment:
Please Note, Package Control v4 actually uses PEP440 versions as it is required for libraries support. This is especially important with regards to pre-release versions.